docs/DotConfig.md

```diff
@@ -2,20 +2,20 @@ Mock Firebolt: mf.config.json(Supported SDK's)
 =======================
 ## Supported SDK's
 
-Mock Firebolt supports these Firebolt SDKs: core, manage, discovery.
-mf.config.json file contains a list of SDK's supported by Mock Firebolt, Out of which `core` is mandatory and other two `manage` and `discovery` are optional.
+Mock Firebolt supports these Firebolt SDKs: core, discovery.
```

Comment:
We do support the manage SDK since the unified file includes its methods/messages.

Reply:
Addressed